TumeKyouenTests/TumeKyouenTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TumeKyouenTests.swift
+//  TumeKyouenTests
+//
+//  Created by 石倉昇 on 2018/03/10.
+//  Copyright © 2018年 noboru. All rights reserved.
+//
+
+import XCTest
+
+class TumeKyouenTests: XCTestCase {
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-ios/blob/d02636fad73e2cfc2f969f4cf8e5308e911c5ff7/TumeKyouenTests/TumeKyouenTests.swift#L12"></span>Lines should not have trailing whitespace.</td>
    </tr>
  </tbody>
</table>